### PR TITLE
Viewdock improvement

### PR DESCRIPTION
**Pull request by SlNPacifist**
Added `Workspace::set_sizer_at` method
`Window` does not keep previous mouse position.

### CODE REVIEW

File: src/prodbg/viewdock/src/area/split.rs [hunk 1]
```diff
@@ -119,18 +118,25 @@ impl Split {
             self.ratios[index + 1] - 0.05
         };
 
-        if res < min {
-            res = min;
+        if ratio < min {
+            ratio = min;
```
Comment (emoon) [inline src/prodbg/viewdock/src/area/split.rs]: Can maybe use min/max functions for this? https://doc.rust-lang.org/std/cmp/fn.min.html

Comment (SlNPacifist) [inline src/prodbg/viewdock/src/area/split.rs]: That'll definitely help :). I'll seek if there is "clamp" function in std, if no - min/max is the best way.

Comment (emoon) [inline src/prodbg/viewdock/src/area/split.rs]: Yeah I looked for clamp but couldn't find one :/ but maybe you have better luck :)

Comment (SlNPacifist) [inline src/prodbg/viewdock/src/area/split.rs]: Could not find either. Replaced with `.min().max()`.

Comment (emoon) [inline src/prodbg/viewdock/src/area/split.rs]: Yeah that is better at least.

**Pull request merged**